### ISSUE TITLE
Signup page: Username/Password validation is not explained

### ISSUE THREAD

**Issue by C-Otto** [TracksApp/tracks#1985]
According to the code, the username must be at least of length 3, the password must have at least 5 characters. While this is somewhat reasonable, (apart from #1984) I don't see any mention of that - even at the time one tries to create an invalid user.

**Comment by ZeiP** [comment]
Yep, we should show at least some kind of decent error message if something goes wrong. Changing to bug, since the error message is missing altogether, not just inaccurate.
